themes/openy_themes/openy_carnation/src/scss/component/_breadcrumbs.scss

```diff
@@ -100,4 +100,20 @@ body.without-banner {
   .breadcrumbs-wrapper {
     position: initial;
   }
+  &.page-node-type-landing-page {
+    .breadcrumbs-wrapper {
+      _margin-top: -225px;
```

Comment:
@retif are You sure that we this will work?
```suggestion
      margin-top: -225px;
```

Reply:
i need to delete that line